resources/prosody-plugins/mod_muc_size.lua

```diff
@@ -7,6 +7,13 @@ local it = require "util.iterators";
 local json = require "util.json";
 local iterators = require "util.iterators";
 local array = require"util.array";
+
+local have_async = pcall(require, "util.async");
+if not have_async then
+    module:log("error", "requires a version of Prosody with util.async");
```

Comment:
what versions don't have this?

Reply:
0.10 and below. The trunk we use 747(really old one) and 0.11 are fine. I'm testing on 18.04 so 0.10 is the default.

Reply:
Ack